poppy/zernike.py

```diff
@@ -309,33 +300,30 @@ def zernike_basis(nterms=15, npix=512, rho=None, theta=None, **kwargs):
 
     Parameters
     -----------
-    nterms : int
-        Number of Zernike terms to return
+    nterms : int, optional
+        Number of Zernike terms to return, starting from piston.
+        (e.g. ``nterms=1`` would return only the Zernike piston term.)
+        Default is 15.
     npix: int
-        Desired pixel diameter for circular pupil. Only used if r and theta are not provided.
+        Desired pixel diameter for circular pupil. Only used if `rho`
+        and `theta` are not provided.
     rho, theta : array_like
-        Image plane coordinates. `rho` should be 1 at the desired pixel radius,
-        `theta` should be in radians
-    noll_normalize : bool
-        As defined in Noll et al. JOSA 1976, the Zernikes are normalized such that
-        the integral of Z[n, m] * Z[n, m] over the unit disk is pi exactly. To omit
-        the normalization constant, set this to False. Default is True.
+        Image plane coordinates. `rho` should be 1.0 at the desired
+        pixel radius, `theta` should be in radians.
```

Comment:
I think I didn't really write this clearly: "should be 1.0 at the desired pixel radius". The way you phrased it above on lines 200-202 for the `zernike` docstring is better, and I think it would make sense to copy that phrasing here?